app-guides/mysql-on-fly.html.markerb

```diff
@@ -59,39 +64,38 @@ Finally, edit the `fly.toml` file that `fly launch` generated.
 For apps using MySQL 8+:
 
 ```toml
-app = "my-mysql"
-kill_signal = "SIGINT"
-kill_timeout = 5
-
-# If copy/paste'ing, adjust this
-# to the region you're deploying to
+# Keep your own app name
+app = 'my-mysql'
+# Keep your own primary region
 primary_region = "bos"
 
+[build]
+  image = 'mysql:8'
+
+[[vm]]
+  cpu_kind = 'shared'
+  cpus = 1
+  memory_mb = 2048
+
+# Add the following sections
 [processes]
-app = """--datadir /data/mysql \
-  --default-authentication-plugin mysql_native_password \
-  --performance-schema=OFF \
-  --innodb-buffer-pool-size 64M"""
+  app = """--datadir /data/mysql \
+    --default-authentication-plugin mysql_native_password \
+    --performance-schema=OFF \
```

Comment:
The two options:

```
    --performance-schema=OFF \
    --innodb-buffer-pool-size 64M
```

...were to get MySQL 8 to work in low-RAM environments (e.g. `256m`), but the docs now assume users will run ~2gb of ram, making these flags unnecessary (probably a great idea).

The easiest thing to do is just remove the flags:

```
  app = """--datadir /data/mysql \
    --default-authentication-plugin mysql_native_password \
    --performance-schema=OFF \
```

The slightly harder version is  making a note to say if you use less than 1gb of RAM, perhaps use those last 2 flags. I'm not entirely sure where that fits best (in comments of fly.toml? as a new paragraph?

Reply:
@fideloper oh was that to get it to deploy before scaling?

I'm confused about `--performance-schema`, should that be removed as well?